site/src/pages/components/menu.explainer.mdx

```diff
@@ -26,101 +26,111 @@ layout: ../../layouts/ComponentLayout.astro
+`<menuitem>` elements are flex items displayed in a row.
 
-Currently, we don’t see a need for this. All menuitems inside a menubar should perform some action
-like opening a sub `<menulist>`, however we will reconsider this restriction if good use cases
-arise.
+This element:
+
+* Provides the implicit [`menubar`](https://w3c.github.io/aria/#menubar) ARIA role.
+* Provides the correct keyboard and focus behavior across child menu items and
+  sub menulists.
 
-For example, a text editor might need to provide a `<menulist>` to provide basic font style adjustments. A group of checkable `<menuitem>` elements to mark content as "Bold", "Italic" or "Underlined", could be provided.  The checked state representing whether the text is currently "Bold" (checked) or not.
+**Content model**: the `<menubar>` element allows the following elements as descendants:
+* `<menuitem>`
+* Anchor `<a>` tags are disallowed in menu bars, due to accessibility concerns around exposing
+  links in ARIA menus, and concerns around the navigation menubar pattern. We enforce this with the
+  content model, and propose default `display: none` styles for anchors that are direct descendants
+  of `<menubar>` elements.
 
-## Grouping `<menuitem>`s together
 
-Our proposal involves grouping `<menuitem>` elements together with the new `<menubar>` and
-`<menulist>` elements, and does not modify the existing [`<menu>`
-element](https://html.spec.whatwg.org/multipage/grouping-content.html#the-menu-element) processing
-model. See https://github.com/openui/open-ui/issues/1194 for more discussion about this.
+## The `<menulist>` element
 
-`<menubar>`
+The `<menulist>` element is another way to group `<menuitem>`s together. This element is a native
+**popover**, and must be invoked by a button or a `<menuitem>` to show what it represents—a stack
+of `<menuitem>`s with default styles that resemble an application drop-down menu.
 
-* Menubar should be used to represent a series of `<menuitem>`s that can then open submenus of commands.
-* In a menubar, a menuitem’s most common purpose is to invoke submenus (menulists).
-* An **in-page** menu that can contain a list of `<menuitem>`.
-* Menuitems within a menubar are displayed horizontally by default.
-* Supports :disabled, disabled attribute.
+This element:
 
-`<menulist>`
+* Provides the implicit [`menu`](https://w3c.github.io/aria/#menu) ARIA role.
+* Has default anchor positioning styles that anchor it to its
+  [implicit anchor](https://drafts.csswg.org/css-anchor-position/#implicit-anchor-element)—the
+  button or menuitem that invoked it.
+* Supports the `:popover-open` pseudo-class, as it is a native **popover**.
+* Supports the
+  [`toggle` event](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/toggle_event), as it
+  is a native **popover**.
+* Can be invoked with the existing `show-popover` command, and the new `show-menu` command. It is
```

Comment:
I'd remove `show-popover` here. We should be pointing to `show-menu` for menu commands. They do differ (see my latest CLs) in some behavior, such as popover stacking.